.github/workflows/release-60_post-crates-release-activities.yml

```diff
@@ -0,0 +1,221 @@
+          echo ""
+          echo "ACTION REQUIRED: Please bump NODE_VERSION manually in polkadot/node/primitives/src/lib.rs"
+          echo "Current version:"
+          grep "NODE_VERSION" polkadot/node/primitives/src/lib.rs || true
+
+      - name: Bump NODE_VERSION for polkadot-parachain and polkadot-omni-node
+        run: |
+          echo "Checking current NODE_VERSION in cumulus..."
+          grep -n "NODE_VERSION" cumulus/polkadot-omni-node/lib/src/nodes/mod.rs || echo "NODE_VERSION not found"
+          echo ""
+          echo "ACTION REQUIRED: Please bump NODE_VERSION manually in cumulus/polkadot-omni-node/lib/src/nodes/mod.rs"
+          echo "Current version:"
+          grep "NODE_VERSION" cumulus/polkadot-omni-node/lib/src/nodes/mod.rs || true
+
+      - name: Check spec_version for runtimes
+        run: |
+          echo "ACTION REQUIRED: Please check if spec_version needs to be bumped for runtimes"
+
+      - name: Move prdocs to release folder
+        shell: bash
+        run: |
+          . ./.github/scripts/release/release_lib.sh
+
+          # Extract release name from branch name (everything after "post-crates-release-")
+          BRANCH_NAME="${{ github.ref_name }}"
+          echo "Branch name: $BRANCH_NAME"
+
+          if [[ "$BRANCH_NAME" =~ post-crates-release-(.+)$ ]]; then
+            RELEASE_FOLDER="${BASH_REMATCH[1]}"
+            echo "Release folder name: $RELEASE_FOLDER"
+
+            # Use the reorder_prdocs helper function
+            reorder_prdocs "$RELEASE_FOLDER"
+          else
+            echo "WARNING: Could not extract release name from branch name: $BRANCH_NAME"
+            echo "Expected format: post-crates-release-<release-name>"
+            exit 1
+          fi
+
+      - name: Replace path dependencies
+        run: |
+          echo "Running replace-all-path-deps.sh..."
+          bash scripts/release/replace-all-path-deps.sh
+          # Show git diff to see what changed
+          git diff --stat
+
+      - name: Remove versions where path deps are present
+        run: |
+          echo "Running delete-versions-if-path-is-present.sh..."
+          if [ -f "scripts/release/delete-versions-if-path-is-present.sh" ]; then
+            bash scripts/release/delete-versions-if-path-is-present.sh
+          else
+            echo "WARNING: delete-versions-if-path-is-present.sh not found"
+          fi
+          # Show git diff to see what changed
+          git diff --stat
+
+      - name: Remove version from umbrella/Cargo.toml
+        run: |
+          echo "Running delete-version-from-umbrella.sh..."
+          bash scripts/release/delete-version-from-umbrella.sh
+          # Show git diff to see what changed
+          git diff --stat
+
+      - name: Install and run Zepter - check issues
+        run: |
+          echo "Installing zepter..."
+          cargo install zepter --version 0.17.0 --locked || echo "zepter installation failed or already installed"
```

Comment:
We don't need to install zepter, it is already pre-installed in the CI image: https://github.com/paritytech/dockerfiles/blob/bb25309d9315f907096b4cccaf43da8f8ecc4f81/ci-unified/Dockerfile#L136